tests/Application/.env

```diff
@@ -22,6 +22,14 @@ DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.
 MAILER_URL=smtp://localhost
 ###< symfony/swiftmailer-bundle ###
 
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+#MESSENGER_TRANSPORT_DSN=doctrine://default
```

Comment:
```suggestion
# MESSENGER_TRANSPORT_DSN=doctrine://default
```
As I'm stand-in for @senghe's laser eye - add a space before :D

Reply:
Done